.github/workflows/docs.yml

```diff
@@ -0,0 +1,29 @@
+name: Publish docs
+
+permissions:
+  contents: write # FIXME: remove this line when the permissions issue is resolved
+  pages: write
+  id-token: write
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - "docs/**"
+      - ".github/workflows/docs.yml"
+  pull_request:
+    branches:
+      - master
+    paths:
+      - "docs/**"
+      - ".github/workflows/docs.yml"
+
+jobs:
+  publish:
+    uses: Alfresco/jekyll-build-tools/.github/workflows/jekyll-publish.yml@main
+    with:
+      working-directory: .
+      publish: true
+      validate-html: true
+      validate-html-args: --ignore-urls "/github.com/,/alfresco.github.io/acs-deployment/"
```

Comment:
try with the new https://github.com/Alfresco/jekyll-build-tools/pull/7 and drop these args